README.md

```diff
@@ -42,62 +42,62 @@ The following packages are needed for other repositories:
 
 > ATTENTION: In addition, we also provide Dockerfiles for Ubuntu and CentOS in `.github/workflows/containers/dockerfiles`. You can use them directly if needed. The following "6: Docker image" has the corresponding usage.
 
-Run `./build-ventus.sh` to automatically build all the programs, but we need to run firstly
-* For developers who want to build `Debug` version for llvm, `export BUILD_TYPE=Debug`, since it's set default to be 'Release'
-* `export POCL_DIR=<path-to-pocl-dir>`, default folder path will be set to be **`<llvm-ventus-parentFolder>`/pocl**
-* `export OCL_ICD_DIR=<path-to-ocl-icd-dir>`, default folder path will be set to be **`<llvm-ventus-parentFolder>`/ocl-icd**
+Before running `./build-ventus.sh` to automatically build all the programs, we need to set the following commands:
+* For developers who want to build `Debug` version for llvm, `export BUILD_TYPE=Debug`, since it's set default to be 'Release'.
+* `export POCL_DIR=<path-to-pocl-dir>`, default folder path will be set to be `<path-to-llvm-ventus>/../pocl`.
+* `export OCL_ICD_DIR=<path-to-ocl-icd-dir>`, default folder path will be set to be `<path-to-llvm-ventus>/../ocl-icd`.
 
-You can dive into `build-ventus.sh` file to see the detailed information about build process
+You can dive into `build-ventus.sh` file to see the detailed information about build process.
 
 ### 3: Bridge icd loader
 
-Run `export VENTUS_INSTALL_PREFIX=<path_to_install>` to set `VENTUS_INSTALL_PREFIX` environment variable(system environment variable recommended)
+Run `export VENTUS_INSTALL_PREFIX=<path_to_install>` to set `VENTUS_INSTALL_PREFIX` environment variable(system environment variable recommended), default folder path will be set to be `<path-to-llvm-ventus>/../install`.
```

Comment:
Typically, this is set to `<path-to-llvm-ventus>/install`.

Reply:
Thanks! I have fixed it.